waiter/src/waiter/util/client_tools.clj

```diff
@@ -1245,6 +1245,14 @@
      (doseq [[_# router-url#] (routers waiter-url#)]
        (is (wait-for #(= etag# (token->etag router-url# token# :cookies cookies#)))))))
 
+(defn has-raven-proxy-instance?
```

Comment:
```suggestion
(defn has-reverse-proxy-instance?
```

Reply:
Resolved this by consistently using raven as the name for the sidecar (see #1437).